packages/wallets/src/features/cfd/screens/CompareAccounts/InstrumentsIconWithLabel.tsx

```diff
@@ -24,9 +23,9 @@ const InstrumentsIconWithLabel = ({ highlighted, icon, isAsterisk, text }: TInst
         >
             {getInstrumentsIcons(isMobile)[icon]}
             <div className='wallets-compare-accounts-trading-instruments__text'>
-                <WalletText align='left' as='p' lineHeight='xs' size='xs' weight={isMobile ? 'normal' : 'bold'}>
+                <Text align='left' as='p' lineHeight='xs' size='xs' weight={isMobile ? 'normal' : 'bold'}>
```

Comment:
can you replace `isMobile` with `!isDesktop`

Reply:
Updated